gnmawsgr/tasks.py

```diff
@@ -276,12 +276,7 @@ def do_glacier_restore(request_id,itemid,path):
     while True:
         try:
             with open(filename,'wb') as fp:
-                try:
-                    key.get_file(fp, cb=partial(download_callback, rq), num_cb=40)
-                except AttributeError:
-                    raven_client.captureException()
-                    logger.error("No Key found in S3. Object values: {0}".format(rq.__dict__))
-                    return
```

Comment:
I think I just understood why this break things. The point of the try statement above the with statement is to either download the file from S3 or fail and then start a Celery task.

I assume putting another try statement inside it is breaking something. I have possibly just got the syntax wrong.